app/main/routes.py

```diff
@@ -84,14 +84,64 @@ def callback():
     session["token_scope"] = access_token_response["scope"]
     session["session_state"] = access_token_response["session_state"]
 
-    return redirect(url_for("main.poc_search"))
+    return redirect(url_for("main.browse"))
 
 
 @bp.route("/accessibility", methods=["GET"])
 def accessibility():
     return render_template("accessibility.html")
 
 
+@bp.route("/browse", methods=["POST", "GET"])
+@access_token_sign_in_required
+def browse():
+    transferring_body_id = request.args.get("transferring_body", None)
+    series_id = request.args.get("series", None)
+    consignment_id = request.args.get("consignment_id", None)
+    query = request.form.get("query", None)
+
+    form = SearchForm()
+    search_results = []
+    search_type = "browse"
+
+    # Create a list to accumulate filter conditions
+    filters = []
+
+    if transferring_body_id:
+        filters.append({"transferring_body_id": transferring_body_id})
+        search_type = "transferring_body"
```

Comment:
like Pavan mentioned, maybe renaming this to 
`browse_type` will be clearer

Reply:
Updated, should be much clearer now removing search logic.